README.md

```diff
@@ -9,75 +9,145 @@ EditURL = "<unknown>/README.jl"
 [![Build Status](https://travis-ci.com/gridap/GridapMakie.jl.svg?branch=master)](https://travis-ci.com/gridap/GridapMakie.jl)
 [![Coverage](https://codecov.io/gh/gridap/GridapMakie.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/gridap/GridapMakie.jl)
 
-# This package is experimental
+## Overview
 
-Things can and will break without deprecation or warning
+The visualization of numerical results is an important part of FE computations. However, the current way of visualizing data from
+[Gridap.jl](https://github.com/gridap/Gridap.jl) computations is to write it to disk in vtu format, thus relying on external software
+such as Paraview. From the idea of visually inspecting data from Julia code directly or to manipulate it with packages of the Julia
+open-source package ecosystem, [GridapMakie.jl](https://github.com/gridap/GridapMakie.jl) is born. As a part of the Google Summer of
+Code 2021 program, GridapMakie adopts [Makie.jl](https://github.com/JuliaPlots/Makie.jl) as a second visualization back-end for
+Gridap.jl simulations. This package is thought as a built-in tool to assess the user in their FE calculations with a smoother workflow
+in a highly intuitive API.
 
-# Examples
+## Installation
 
-```@example README
-using Gridap, GridapMakie, Makie
+According to Makie's guidelines, it is enough to install one of its backends, e.g. GLMakie. Additionally, Gridap provides the plot objects
+to be visualized
 
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (20,10)))
-scene = plot(model)
+```julia
+julia> ]
+julia> add Gridap, GridapMakie, GLMakie
 ```
 
-![](_readme/images/2d.png)
+## Examples
 
-```@example README
-data = GridapMakie.demo_data(spacedim=1, valuetype=Float64)
-scene = plot(data.u, data.model)
-```
+First things first, we shall be using the three packages
 
-![](_readme/images/plot_1d_Scalar.png)
+````@example README
+using Gridap, GridapMakie, GLMakie
+````
 
-```@example README
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (10,10)))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = Float64,
-    conformity=:H1, model=model)
-f(pt) = sin(pt[1])*cos(pt[2])
-u = interpolate(V, f)
-scene = mesh(u, model, color=:yellow)
-```
+### 2D Plots
 
-![](_readme/images/mesh_2d_Scalar.png)
+Then, let us consider a simple, 2D simplexified cartesian mesh Ω
 
-```@example README
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (10,10)))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = Float64,
-    conformity=:H1, model=model)
-f(pt) = sin(pt[1])*cos(pt[2])
-u = interpolate(V, f)
-scene = wireframe(u, model)
-```
+````@example README
+domain = (0, 1, 0, 1)
+cell_nums = (10, 10)
+model = CartesianDiscreteModel(domain, cell_nums) |> simplexify
+Ω = Triangulation(model)
+````
 
-![](_readme/images/wireframe_2d_Scalar.png)
+The visualization of Ω along with the edges of its faces and its vertices
```

Comment:
Replace

"The visualization of Ω along with the edges of its faces and its vertices"

by 

"The visualization of the vertices and edges of Ω can be achieved as follows" ???